doc/sphinx/language/gallina-extensions.rst

```diff
@@ -874,7 +869,7 @@ Sections create local contexts which can be shared across multiple definitions.
       :name: Let Fixpoint
       :undocumented:
 
-   .. cmdv:: Let CoFixpoint @ident @cofix_body {* with @cofix_body}
+   .. cmdv:: Let CoFixpoint @ident @fix_body {* with @fix_body}
```

Comment:
Did I understand correctly that this change should be cancelled after #11103 is merged?

Reply:
We should not wait for the other 2 PRs.  #11103 is not so simple, it may take several days or more to get it merged.  There's already so much to do to get the correct grammar into the doc.  Adding more delay is a very bad idea.  We can easily adjust for their changes later.
- The grammar will continue to change independently of the rsts--until and unless we require that PRs modifying mlgs also update the doc_grammar files and rsts.
- It's generally very easy to update the doc_grammar info to track grammar changes.  I've done it a dozen times, usually it takes little or no thought and only a couple minutes.
- #11103 has many changed lines.  The easy way to figure out how the grammar has changed is to run the changed files through doc_grammar--one of the design goals of the tool.  It's much, much harder to do that by reading the mlg files.

> Did I understand correctly that this change should be cancelled after #11103 is merged?

Maybe.  Merge #11103 first.  But don't make this PR wait for that.

Also keep in mind that we haven't started revising the extensions chapter or the commands.